randovania/games/fusion/exporter/hint_namer.py

```diff
@@ -0,0 +1,145 @@
+from __future__ import annotations
```

Comment:
it'd be nice to have tests for this file.

Reply:
I unfortunately cannot find any comparable tests in other games.
What approximately needs to be done in the test and maybe I can take it from there

Reply:
just feed inputs for the important functions (format_resource_is_starting, format_guaranteed_resource, format_location_hint, format_region, format_area, format_player, format_joke) and check for a result